src/test/crimson/test_errorator.cc

```diff
@@ -18,6 +19,15 @@ struct errorator_test_t : public seastar_test_suite_t {
       }
     });
   }
+  ertr::future<> test_parallel_for_each() {
+    std::vector<int> loops = {1, 2, 3};
+    return ertr::parallel_for_each(loops, [](int i) {
+      using namespace std::chrono_literals;
+      return seastar::sleep(1s).then([] {
```

Comment:
i'd avoid `sleep` in test unless necessary. it just slows down the unit test.

Reply:
Yeah, `1s` is probably too much. However, the time domain is pretty good, intuitive way of illustrating the `parallel` part of the naming.